lib/utils/webpack.config.js

```diff
@@ -0,0 +1,197 @@
+    } else {
+        result = false
+    }
+
+    return result
+}
+
+module.exports = function(program, directory, stage, webpackPort, routes) {
+    webpackPort = webpackPort || 1500;
+    routes = routes || [];
+    function output() {
+        switch(stage) {
+        case "develop":
+            return {
+                path: directory,
+                filename: 'bundle.js',
+                publicPath: `http://${program.host}:${webpackPort}/`
+            }
+        case "production":
+            return {
+                filename: "bundle.js",
+                path: directory + "/public"
+            }
+        case "static":
+            return {
+                path: directory + "/public",
+                filename: "bundle.js",
+                libraryTarget: 'umd'
+            }
+        }
+    }
+
+    function entry() {
+        switch(stage) {
+        case "develop":
+            return [
+                `${__dirname}/../../node_modules/webpack-dev-server/client?${program.host}:${webpackPort}`,
+                `${__dirname}/../../node_modules/webpack/hot/only-dev-server`,
+                `${__dirname}/web-entry`
+            ]
+        case "production":
+            return [
+                `${__dirname}/web-entry`
+            ]
+        case "static":
+            return [
+                `${__dirname}/static-entry`
+            ]
+        }
+    }
+
+    function plugins() {
+        switch(stage) {
+        case "develop":
+            return [
+                new webpack.HotModuleReplacementPlugin(),
+                new webpack.DefinePlugin({
+                    "process.env": {
+                        NODE_ENV: JSON.stringify(process.env.NODE_ENV ? process.env.NODE_ENV : "development")
+                    },
+                    __PREFIX_LINKS__: program.prefixLinks
+                })
+            ]
+        case "production":
+            return [
+                new webpack.IgnorePlugin(/^\.\/locale$/, /moment$/),
+                new webpack.DefinePlugin({
+                    "process.env": {
+                        NODE_ENV: JSON.stringify(process.env.NODE_ENV ? process.env.NODE_ENV : "production")
+                    },
+                    __PREFIX_LINKS__: program.prefixLinks
+                }),
+                new webpack.optimize.DedupePlugin(),
+                new webpack.optimize.UglifyJsPlugin()
+            ]
+        case "static":
+            return [
+                new StaticSiteGeneratorPlugin('bundle.js', routes),
+                new webpack.DefinePlugin({
+                    "process.env": {
+                        NODE_ENV: JSON.stringify(process.env.NODE_ENV ? process.env.NODE_ENV : "production")
+                    },
+                    __PREFIX_LINKS__: program.prefixLinks
+                })
+            ]
+        }
+    }
+
+    function resolve() {
+        return {
+            extensions: ['', '.js', '.jsx', '.cjsx', '.coffee', '.json', '.less', '.toml', '.yaml'],
```

Comment:
PR for the fix is here: https://github.com/lewie9021/webpack-configurator/pull/10